### PR TITLE
Add Yola.set_site_url method

### PR DESCRIPTION
**Pull request by kosmos342**
part of https://github.com/yola/bottomline/issues/224

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,7 @@
+0.7.0
```
Comment (serhii-karelov) [inline CHANGELOG.rst:1]: Shouldn't it be 0.6.3, since we do not bump major version in this repo?

**Review by serhii-karelov — APPROVED** [review]
lgtm

File: CHANGELOG.rst (hunk 1 above)
Comment (kosmos342) [reply CHANGELOG.rst:1]: I think that we use [Semantic Versioning](https://semver.org/) everywhere now

**Pull request merged**